packages/viewport/tsconfig.json

```
@@ -0,0 +1,15 @@
+{
+	"$schema": "https://json.schemastore.org/tsconfig.json",
+	"extends": "../../tsconfig.base.json",
+	"compilerOptions": {
+		"rootDir": "src",
+		"declarationDir": "build-types",
+		"types": [ "gutenberg-env" ]
+	},
+	"references": [
+		{ "path": "../compose" },
+		{ "path": "../data" },
+		{ "path": "../element" }
+	],
+	"include": [ "src/**/*" ]
```

We don't need wildcard here.

```suggestion
	"include": [ "src" ]
```